disperser/batcher/batcher.go

```diff
@@ -312,10 +312,12 @@ func (b *Batcher) updateConfirmationInfo(
 
 		if status == disperser.Confirmed {
 			if _, updateConfirmationInfoErr = b.Queue.MarkBlobConfirmed(ctx, metadata, confirmationInfo); updateConfirmationInfoErr == nil {
+				b.EncodingStreamer.RemoveEncodedBlob(metadata)
```

Comment:
what is the chance of memory leak, when it falls into the last else statement part. CAn we just remove it regardless of the "if" condition

Reply:
in addition to that. the memory load on the batcher for 600sec batch interval would be 10MB/s*600 = 6GB. It looks manageable

Reply:
Sg, removing regardless of the if condition seems benign